app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -1,4 +1,5 @@
 /* global Runtime */
+import { addSnackbarItem } from '../../Snackbar';
```

Comment:
No need to import here. This is an initializer that runs in a global scope

Reply:
Oh! Sorry about that! eslint was showing an error so I thought of importing it. I wasn't aware that this runs in a global scope (I did see the comment up top, but I wasn't certain if I had to suppress the eslint error)

Anyway, I removed it and suppressed the eslint error :+1: Sorry again!

Reply:
Oh sorry, I forgot that you were going to run into a eslint check because `addSnackbarItem` wasn't declared in this file. The way to properly fix the eslint is to add it to that annotation at the very top so it would look like this:

```js
/* global Runtime, addSnackbarItem */
```

Reply:
Would you like me to change the `// eslint-disable-next-line`? And replace it with the fix you suggested above?